azure_jumpstart_hcibox/artifacts/PowerShell/New-HCIBoxCluster.ps1

```diff
@@ -0,0 +1,1632 @@
+$mgmtMac = New-ManagementVM -Name $($HCIBoxConfig.MgmtHostConfig.Hostname) -VHDXPath "$HostVMPath\GUI.vhdx" -VMSwitch $InternalSwitch -HCIBoxConfig $HCIBoxConfig
+Set-MGMTVHDX -VMMac $mgmtMac -HCIBoxConfig $HCIBoxConfig
+
+# Create the HCI host node VMs
+Write-Host "[Build cluster - Step 4/10] Creating HCI node VMs (AzSHOSTx)..." -ForegroundColor Green
+foreach ($VM in $HCIBoxConfig.NodeHostConfig) {
+    $mac = New-HCINodeVM -Name $VM.Hostname -VHDXPath $hcipath -VMSwitch $InternalSwitch -HCIBoxConfig $HCIBoxConfig
+    Set-HCINodeVHDX -HostName $VM.Hostname -IPAddress $VM.IP -VMMac $mac  -HCIBoxConfig $HCIBoxConfig
+}
+    
+# Start Virtual Machines
+Write-Host "[Build cluster - Step 5/10] Starting VMs..." -ForegroundColor Green
+Write-Host "Starting VM: $($HCIBoxConfig.MgmtHostConfig.Hostname)"
+Start-VM -Name $HCIBoxConfig.MgmtHostConfig.Hostname
+foreach ($VM in $HCIBoxConfig.NodeHostConfig) {
+    Write-Host "Starting VM: $($VM.Hostname)"
+    Start-VM -Name $VM.Hostname
+}
+
+#######################################################################################
+# Prep the virtualization environment
+#######################################################################################
+Write-Host "[Build cluster - Step 6/10] Configuring host networking and storage..." -ForegroundColor Green
+# Wait for AzSHOSTs to come online
+Test-AllVMsAvailable -HCIBoxConfig $HCIBoxConfig -Credential $localCred
+Start-Sleep -Seconds 60
+
+# Format and partition data drives
+Set-DataDrives -HCIBoxConfig $HCIBoxConfig -Credential $localCred
+    
+# Configure networking
+Set-NICs -HCIBoxConfig $HCIBoxConfig -Credential $localCred
+    
+# Restart Machines
+Restart-VMs -HCIBoxConfig $HCIBoxConfig -Credential $localCred
+    
+# Wait for AzSHOSTs to come online
+Test-AllVMsAvailable -HCIBoxConfig $HCIBoxConfig -Credential $localCred
+
+# Create NAT Virtual Switch on AzSMGMT
+New-NATSwitch -HCIBoxConfig $HCIBoxConfig
+
+# Configure fabric network on AzSMGMT
+Set-FabricNetwork -HCIBoxConfig $HCIBoxConfig -localCred $localCred
+
+#######################################################################################
+# Provision the router, domain controller, and WAC VMs and join the hosts to the domain
+#######################################################################################
+# Provision Router VM on AzSMGMT
+Write-Host "[Build cluster - Step 7/10] Build router VM..." -ForegroundColor Green
+New-RouterVM -HCIBoxConfig $HCIBoxConfig -localCred $localCred
+
+# Provision Domain controller VM on AzSMGMT
+Write-Host "[Build cluster - Step 8/10] Building Domain Controller VM..." -ForegroundColor Green
+New-DCVM -HCIBoxConfig $HCIBoxConfig -localCred $localCred -domainCred $domainCred
+
+# Provision Admincenter VM
+# Write-Host "[Build cluster - Step 9/12] Building Windows Admin Center gateway server VM... (skipping step)" -ForegroundColor Green
+#New-AdminCenterVM -HCIBoxConfig $HCIBoxConfig -localCred $localCred -domainCred $domainCred
```

Comment:
Can this be removed if not used? Or should the Admin Center VM be installed by this command?